src/backend/src/services/change-requests.services.ts

```diff
@@ -384,6 +383,14 @@ export default class ChangeRequestsService {
       );
     }
 
+    const reviewedProject = wbsElement.changeRequests.filter((cr) => cr.dateReviewed === null);
```

Comment:
make this reviewedProjects with an s because this is technically multiple projects

Reply:
also it should be unreviewedChangeRequests